docs/guides/options.md

```diff
@@ -162,6 +162,43 @@ Prevents the player from running the autoSetup for media elements with `data-set
 
 > **Note**: this must be set globally with `videojs.options.autoSetup = false` in the same tick as videojs source is loaded to take effect.
 
+### `breakpoints`
+
+> Type: `boolean|Array`, Default: `false`
+
+Set layout breakpoints that will toggle classes on the player and adjust the player UI based on the player's dimensions.
+
+By default, no breakpoints are supported, but passing `true` will set some sensible default breakpoints:
+
+Class                | Width Range
+---------------------|------------
+`vjs-layout-tiny`    | 0-210
+`vjs-layout-x-small` | 211-320
+`vjs-layout-small`   | 321-425
+`vjs-layout-medium`  | 426-768
+`vjs-layout-large`   | 769-1440
+`vjs-layout-x-large` | 1441-2560
+`vjs-layout-huge`    | 2561+
+
+These can be overridden by passing an **ordered** array that looks like this:
```

Comment:
Gotcha, that could make sense. We define the order by class name and then it can be configured via an object where ordering does not matter. I like it. 👍